packages/utils/upgrade/resources/codemods/5.0.0/sqlite3-to-better-sqlite3.json.ts

```diff
@@ -30,8 +30,7 @@ const transform: modules.runner.json.JSONTransform = (file, params) => {
   });
 
   if (removed && !j.has('dependencies.better-sqlite3')) {
-    // TODO check this version when releasing V5
-    j.set('dependencies.better-sqlite3', '9.4.3');
+    j.set('dependencies.better-sqlite3', '11.3.0');
```

Comment:
It's a bit strange to change existing codemods. Shouldn't this be part of a new codemod to upgrade to 5.1.0 with node 22 support ? (it should def trigger a minor release)

Reply:
Yep, definitely will make it a minor release.

As far as changing existing codemods, yeah, I wasn't sure about that. I can add a new codemod for 5.1.0 instead since the upgrade tool already supports that.

Reply:
i don’t see why it’s strange to change an existing codemod; the codemod's purpose is not changing after all

Reply:
Glad we added support for codemods on minor/patch versions :slightly_smiling_face:

Reply:
The whole point @maccomaccomaccomacco is that you can run the upgrade tool when you are already on v5. if we change that existing codemod and someone already has v5 they won't benefit from this change. when we release it, It's a shame